README.md

```diff
@@ -303,6 +303,93 @@ The credit card charge result may contains `redirect_url` for 3DS authentication
 For full example on Credit Card 3DS transaction refer to:
 - [Flask App examples](/examples/flask_app) that implement Snap & Core Api
 
+### 2.2.D Subscription API
+You can see some Subscription API examples [here](examples/subscription), [Subscription API Docs](https://api-docs.midtrans.com/#subscription-api)
+
+```python
+# Create Subscription API instance
+subscription = midtransclient.CoreApi(
```

Comment:
don't use `subscription` as the object name it's confusing, just name it as `core_api` because you are actually creating instance of CoreApi class.